core/google/cloud/_helpers.py

```diff
@@ -22,27 +22,24 @@
 
 import calendar
 import datetime
-import json
 import os
 import re
-import socket
 from threading import local as Local
 
+import google.auth
 from google.protobuf import timestamp_pb2
-try:
-    from google.appengine.api import app_identity
-except ImportError:
-    app_identity = None
-try:
+import google_auth_httplib2
+
+try:  # pragma: NO COVER
     import grpc
-except ImportError:  # pragma: NO COVER
+    from google.auth.transport.grpc import AuthMetadataPlugin
```

Comment:
1) It's just not in the requirements, nor do I think it should be in the requirements for grpc.
2) Ah, hmmm. Do you have strong feels on how this should be handled or are you okay leaving as-is?